src/Beman/Optional26/tests/optional_constexpr.t.cpp

```diff
@@ -0,0 +1,1261 @@
+// src/Beman/Optional26/tests/optional_constexpr.t.cpp                -*-C++-*-
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <Beman/Optional26/optional.hpp>
+#include <Beman/Optional26/tests/test_types.hpp>
+
+#include <functional>
+#include <ranges>
+#include <tuple>
+
+#include <gtest/gtest.h>
+
+TEST(OptionalConstexprTest, TestGTest) { ASSERT_EQ(1, 1); }
```

Comment:
I don't know if we need to add this test in each file. Do you want to propose this as a convention?

Reply:
It's actually a hold-over from the template I use where it's actually ASSERT_EQ(0,1) so I don't spend a lot of time working on tests that aren't being built or run. 

Deleting it would be fine once it's served its purpose. 

Adding a note to work out a lint check, on my long list.